specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -18,6 +18,72 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupUsageSummaries": {
+      "get": {
+        "tags": [
+          "BackupUsageSummaries_CRR"
+        ],
+        "description": "Fetches the backup management usage summaries of the vault.",
+        "operationId": "BackupUsageSummariesCRR_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "OData filter options.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "description": "skipToken Filter.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BackupManagementUsageList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/NewErrorResponse"
+            }
+          }
+        },
+        "x-ms-odata": "#/definitions/BMSBackupSummariesQueryObject",
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
this will never have pages?

Reply:
How is there a skipToken if there is no paging?

Reply:
Updated the nextLinkName

Reply:
Hi @markcowl. Since this is not a new API that is being added but a replication of the API backupUsageSummaries(we are using this version to route the API to passive stamp for cross region restore) which is already checked in the version 2021-03-01 in the same package, changing any attribute in this version 2018-12-20 will lead to SDK generation issues.

Reply:
This is the original API definition : https://github.com/Azure/azure-rest-api-specs/blob/f15b4f3299df66c755156aaea2c3f0c64f1840bb/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-03-01/bms.json#L423

Reply:
We can take this change only when we upgrade the active stamp version 2021-03-01 else it will break the existing customers.  Citing these reasons I am reverting the latest commits to add the nextLink.